homeassistant/components/ping/device_tracker.py

```diff
@@ -108,7 +115,7 @@ async def async_setup_scanner(
             for (dev_id, ip) in config[CONF_HOSTS].items()
         ]
 
-        async def async_update(now):
+        async def async_update(now: datetime) -> None:
```

Comment:
```suggestion
        async def async_update() -> None:
```

Reply:
This doesn't work as `async_track_point_in_utc_time()` send a `now` parameter so we need to keep it.